testsuites/CBLTester/upgrade_tests/test_cbl_upgrade.py

```diff
@@ -352,3 +355,69 @@ def _upgrade_db(args):
+    cbl_db = "upgrade_db" + str(time.time())
+    # Create CBL database
+    db = Database(base_url)
+
+    log_info("Creating a Database {}".format(cbl_db))
+    source_db = db.create(cbl_db)
+
+    upgraded_liteserv_version = params_from_base_suite_setup["upgraded_liteserv_version"]
+    liteserv_platform = params_from_base_suite_setup["liteserv_platform"]
+    liteserv_host = params_from_base_suite_setup["liteserv_host"]
+    liteserv_port = params_from_base_suite_setup["liteserv_port"]
+    debug_mode = params_from_base_suite_setup["debug_mode"]
+    device_enabled = params_from_base_suite_setup["device_enabled"]
+    community_enabled = params_from_base_suite_setup["community_enabled"]
+    testserver = params_from_base_suite_setup["testserver"]
+    test_name_cp = params_from_base_suite_setup["test_name_cp"]
+    channels_sg = ["ABC"]
+    # Create CBL database
+    db.create_bulk_docs(20, "cbl-upgrade-docs", db=source_db, channels=channels_sg)
+
+    time.sleep(2)
+    base_cbl_doc_count = db.getCount(source_db)
+    testserver2 = TestServerFactory.create(platform=liteserv_platform,
+                                           version_build=upgraded_liteserv_version,
+                                           host=liteserv_host,
+                                           port=liteserv_port,
+                                           community_enabled=community_enabled,
+                                           debug_mode=debug_mode)
+    log_info("Downloading TestServer ...")
+    # Download TestServer app
+    testserver2.download()
+
+    log_info("Installing TestServer ...")
+    # Install TestServer app
+    if device_enabled:
+        testserver2.install_device()
+    else:
+        testserver2.install()
+
+    if device_enabled:
+        testserver2.start_device("{}/logs/{}-{}-{}.txt".format(RESULTS_DIR, type(testserver).__name__,
+                                                               test_name_cp,
+                                                               datetime.datetime.now()))
+    else:
+        testserver2.start("{}/logs/{}-{}-{}.txt".format(RESULTS_DIR, type(testserver).__name__,
+                                                        test_name_cp,
+                                                        datetime.datetime.now()))
+
+    db = Database(base_url)
+    source_db = db.create(cbl_db)
+    upgraded_cbl_doc_count = db.getCount(source_db)
+    cbl_doc_ids = db.getDocIds(source_db)
+    cbl_db_docs = db.getDocuments(source_db, cbl_doc_ids)
+    assert upgraded_cbl_doc_count == base_cbl_doc_count, " Number of docs count is not matching"
+    # Create few more docs and make sure the after upgrade we still create docs
+    db.create_bulk_docs(2, "cbl-upgrade-docs_new", db=source_db, channels=channels_sg)
+    upgraded_cbl_doc_count = db.getCount(source_db)
+    assert upgraded_cbl_doc_count > base_cbl_doc_count, " Number of docs count is not matching"
+    log_info("*" * 20)
+    log_info(cbl_db_docs)
```

Comment:
it is unnecessarily add too much logging . Can you remove this if you add it for debugging purpose for both log_info?